pupepat/plot.py

```diff
@@ -31,3 +36,29 @@ def plot_best_fit_ellipse(output_filename, data_cutout, best_fit_model, header):
                 'FOCDMD: {focdmd: 0.3f} mm'.format(focdmd=header['FOCDMD']), color='white')
     pyplot.tight_layout()
     pyplot.savefig(output_filename)
+
+
+def plot_quiver(data, focus_surface, output_plot, dpi=200):
+    pyplot.clf()
+    pyplot.grid(True)
+    pyplot.gcf().set_size_inches(11, 10, forward=True)
+    fit_evaluator = SurfaceFitter()
+    X, Y = np.meshgrid(1.1 * np.linspace(min(data['M2ROLL']), max(data['M2ROLL']), 10 * dpi),
```

Comment:
Maybe lowercase these vars? Also, a couple magic numbers

Reply:
Which variables, the X and Y?

Reply:
Yep, just cause usually variables are lower case

Reply:
Fair enough. Numpy often uses uppercase variables for 2d arrays and lower case for 1d arrays. I'll just call it x_grid or something.

Reply:
Also- what is the 1.1 for?